package.json

```diff
@@ -25,7 +25,8 @@
     "react": "15.4.2"
   },
   "dependencies": {
-    "lodash": "^4.17.4"
+    "lodash": "^4.17.4",
```

Comment:
It's purely an issue of file size. According to https://lodash.com, the library you are pulling in here is 24kB compressed. That's a big penalty for code that runs in the browser, and also pulls in other heavy libraries. If all you need are a couple of lodash methods, you could try to find micro-libraries that implement them and are around 1kB each. I'm not sure if WebPack is smart enough to strip out unused code. Not a super important issue though, just a hint.